grakn-test/src/test/java/ai/grakn/test/engine/backgroundtasks/TaskStateGraphStoreTest.java

```diff
@@ -74,78 +76,53 @@ public void testStoreRetrieve() {
     }
 
     @Test
-    public void testStoreInvalid() {
-        String id = stateStorage.newState(null, null, null, null, 0, null);
-        assertNull(id);
-
-        id = stateStorage.newState(null, this.getClass().getName(), Instant.now(), true, 10000, new JSONObject());
-        assertNull(id);
-
-        id = stateStorage.newState(TestTask.class.getName(), null, Instant.now(), true, 10000, new JSONObject());
-        assertNull(id);
-
-        id = stateStorage.newState(TestTask.class.getName(), this.getClass().getName(), null, true, 10000, new JSONObject());
-        assertNull(id);
-
-        id = stateStorage.newState(TestTask.class.getName(), this.getClass().getName(), Instant.now(), null, 10000, new JSONObject());
-        assertNull(id);
-    }
-
-    @Test
     public void testUpdate() {
```

Comment:
In an effort to make tests more clear. Test update what? What in the task are you checking is being updated?